examples/playlist/ads.js

```diff
@@ -123,21 +121,18 @@ Ads.prototype.onAdEvent = function(event) {
 };
 
 Ads.prototype.onPlaylistItemClick = function(event) {
-  if (!this.initialized) {
-    this.init();
-  }
   if (!this.linearAdPlaying) {
+    if (!this.initialized) {
+      // Handles the case where the user loads the page and clicks a playlist item
+      // immediately, never clicks the play button on the player.
```

Comment:
nit: ...loads the page, clicks a playlist item immediately and never clicks...

Reply:
Fixed.